src/utils/Logger.js

```diff
@@ -0,0 +1,64 @@
+export const debugOn = new URLSearchParams(window.location.search).has("debug");
```

Comment:
Do we want this to be enable by default when running in e.g. local-stack? Then we could change this line to the following:

```suggestion
export const debugOn = process.env.NODE_ENV === "development" || new URLSearchParams(window.location.search).has("debug");
```

Reply:
If we make this change then I think we could also remove `isEnabled` from the `Logger` constructor.

i.e. if logs are enabled in development mode + whenever ?debug is set, then we shouldn't need to override that setting in the Logger constructor (we could just read `debugOn` directly from this file).

Reply:
Personally I would like to define `isEnabled` locally and don't use it in constructor, like you're suggesting and like I did before. I guess it's a matter of how much we want to deviate from the original Logger class.

Reply:
fixed in [31e3fdf](https://github.com/whereby/jslib-media/pull/51/commits/31e3fdf6a85d6c5e71c31858f7b06aaff31ab0c2)